docs/user-guide/map-templates.md

```diff
@@ -0,0 +1,66 @@
+# Map Templates
+*******************
+
+The **Map Templates** is an extension added to the context to browse the map template like the *WMC templates* and the *MapStore JSON*. The <img src="../img/button/map-templates-button.jpg" class="ms-docbutton"/> button, present in *Burger Menu* <img src="../img/button/burger.jpg" class="ms-docbutton" />, provide to the user the list of the available templates. 
+
+<img src="../img/map-templates/map-templates-panel.jpg" class="ms-docimage"/>
+
+For each template in the *Map Templates* list the following buttons are displayed:
+
+<img src="../img/map-templates/map-templates-buttons.jpg" class="ms-docimage" style="max-width:500px;"/>
+
+* The **Replace** button <img src="../img/button/replace-button.jpg" class="ms-docbutton" /> allows the user to replace the current map context with the template, as follows:
+
+<img src="../img/map-templates/replace-map.gif" class="ms-docimage"/>
+
+* The **Add Template** button <img src="../img/button/add_to_map_button.jpg" class="ms-docbutton" /> allows the user to add the template to current map context, as follows:
+
+<img src="../img/map-templates/add-templates.gif" class="ms-docimage"/>
+
+* The **Add to favorites** button <img src="../img/button/favourites-button.jpg" class="ms-docbutton" /> allows the user to add the template to favorites on top of the list
+
+## Enabling the Map templates in a context
+
+The *Map templates* extension is enabled by the admin in the [Application Context](application-context.md) wizard. In particular, this happens in the [third step](application-context.md#configure-plugins) of the wizard and after the extension has been added to the *Enabled Plugins* list.
```

Comment:
The *Map templates* extension is enabled by the admin in the [Application Context](application-context.md) wizard. In particular, this is possible in the [third step](application-context.md#configure-plugins) of the wizard and after the extension is added to the *Enabled Plugins* list.